pkg/pgsql/server/query_machine.go

```diff
@@ -211,6 +211,8 @@ func (s *session) QueriesMachine() (err error) {
 				s.ErrorHandle(err)
 				waitForSync = true
 			}
+		case fm.FlushMsg:
+			// there is no buffer to be flushed
```

Comment:
Do we need to send any kind of message back?

Reply:
No, this message mean to empty the server buffer, but in our case there is no buffer since at each command is returned immediatly a response